packages/esm-patient-orders-app/src/components/orders-details-table.component.tsx

```diff
@@ -430,12 +433,21 @@ function OrderBasketItemActions({
   const alreadyInBasket = items.some((x) => x.uuid === orderItem.uuid);
 
   const handleViewEditClick = useCallback(() => {
-    // openOrderForm({ order: orderItem });
+    if (orderItem.type === 'drugorder') {
```

Comment:
> It also appears that modifying an order creates a new order on top of the existing one. Is this intentional?

You can only modify an order before it's submitted. Once it's been submitted, "modifications" are done via a new order that replaces the previous one.

> Shouldn't the test type be read-only when editing?

Yes. This is actually a (minor) safety issue.

Reply:
@denniskigen this was a little bit out of scope for this PR but decided to fix it anyways in the recent commit. I've tested this with both medications and labs and all is working as expected. There's some different in behaviour with revised medication being annotated with (REVISED) but labs isn't even though confirming on postman and UI, it works as intended.

Reply:
I just tested the flow for modifying a lab order, and I can confirm that the existing order was modified without creating a new one. So that's good. Things that still need to be worked on:

- I don't expect the `Test type` to be an editable field in the form when modifying an order. Doesn't editing the test type mean you want to create a new order anyway?
- The workspace title should change to reflect the action, e.g. `Modify lab order` when modifying a lab order, or `Modify drug order` when modifying a drug order.

Reply:
>You can only modify an order before it's submitted. Once it's been submitted, "modifications" are done via a new order that replaces the previous one.

@ibacher I could be wrong but the [ImmutableOrderInterceptor](https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/api/db/hibernate/ImmutableOrderInterceptor.java#L25-L26) seems to give a pass at props like: `fulfillerStatus`, `fulfillerComment`, `accessionNumber` etc. (I didn't find any subclasses)

Reply:
@denniskigen regarding more changes to the edit orders, am going to add note to [this ticket](https://openmrs.atlassian.net/browse/O3-2813) and submit a follow up PR so this doesn't drag. The main purpose for this was for add lab results and being able to view them